.dockerignore

```diff
@@ -1,3 +0,0 @@
-venv
-*.coverage
-coverage_reports
```

Comment:
First line seems like a legacy from project inception. Any reason you added lines 2 and 3 @tomerm-iguazio?

Reply:
Refer to:
https://github.com/github/gitignore/blob/main/Python.gitignore
`.coverage` is for Coverage.py artifacts.
`coverage_reports` - not sure about this one.
`venv` is also very common.

Reply:
Yeah, I know what they are, I just want to know if @tomerm-iguazio added them for a specific reason.

Reply:
I added them for generated coverage reports

Reply:
If we are not building docker image anymore, it is irrelevant.
I agree that It is required for gitignore file.